core/src/main/java/org/geysermc/geyser/translator/protocol/java/level/JavaLevelChunkWithLightTranslator.java

```diff
@@ -318,7 +314,7 @@ public void translate(GeyserSession session, ClientboundLevelChunkWithLightPacke
                             session.getBlockMappings().getBedrockWater().getRuntimeId());
 
                     layers = new BlockStorage[]{ layer0, new BlockStorage(BitArrayVersion.V1.createArray(BlockStorage.SIZE, layer1Data), layer1Palette) };
-                } else if (waterloggedPaletteIds.isEmpty() && extendedCollision) {
```

Comment:
?

Reply:
by the time that condition is reached, extendedBlockCollisions is always true